fe/fe-core/src/main/java/com/starrocks/statistic/StatisticUtils.java

```diff
@@ -100,11 +100,13 @@ public static ConnectContext buildConnectContext() {
         // but QeProcessorImpl::reportExecStatus will check query id,
         // So we must disable report query status from BE to FE
         context.getSessionVariable().setEnableProfile(false);
+        context.getSessionVariable().setEnableLoadProfile(false);
```

Comment:
don't enable in statistics connection? may be you need another buildConnectContext method